discordBot.js

```diff
@@ -43,6 +43,47 @@ class DiscordBot {
   stop() {
     return this.client.destroy();
   }
+
+  // send files to discord channel given the file path and channel
+  sendFile(filePath, channelId){
+    this.client.on('ready', () => {
+      var attachment = new MessageAttachment(filePath);
+      this.client.channels.cache.get(channelId).send(attachment);
+      return 'success';  
```

Comment:
we should return the response object instead of the 'success' string from our library, so it's up the develop to decide what should be return to user. and it's easier for the developer to debug if we return the response object

Reply:
ok did that now let me know if on the other issues.